### PR TITLE
Adds ephedrine and menthol to the nanomed/porta nanomed

### PR DESCRIPTION
**Pull request by NightmareChamillian**
## About the PR 
This adds an ephedrine and menthol autoinjector, as well as bottle objects to the game. It also buffs the number of embalming fluid syringes by 1, since I've found they run out pretty fast

## Why's this needed? 

Since menthol pill bottles are usually alongside salicylic acid pill bottles in the game, it would make sense that you can also get menthol for fast use in the same situations that you can salicylic acid. Also, since ephedrine is different enough from epinephrine, and it's a really, really useful medication, I added it to the nanomed. The only real concern with it is the mass production of meth, which I addressed by limiting the bottles to be only three, on par with atropine and the nastier medbay chems. I still can further reduce it to only 2 if there's any further concerns, though.

## Changelog

```changelog
(u)NightmareChamillian
(*)Ephedrine and menthol are now available in the porta- and normal nanomed
```

### CODE REVIEW

**Comment by Flaborized** [comment]
This is a very minor nitpick but the description for the ephedrine bottle explains that it's a stimulant twice in a row

> "A small bottle containing ephedrine, **a stimulant**. Not to be confused with epinephrine, ephedrine is a general purpose medication and **stimulant**..."

**Comment by NightmareChamillian** [comment]
> This is a very minor nitpick but the description for the ephedrine bottle explains that it's a stimulant twice in a row
bad grammar is still bad grammar; fixed!

**Comment by Cthucky** [comment]
Might want to whitelist ephedrine for hyposprays while you're at it.

File: code/modules/vending/vending.dm [hunk 1]
```diff
@@ -1151,10 +1151,12 @@
 		product_list += new/datum/data/vending_product(/obj/item/reagent_containers/glass/bottle/calomel, 3)
 		product_list += new/datum/data/vending_product(/obj/item/reagent_containers/glass/bottle/antitoxin, 5)
 		product_list += new/datum/data/vending_product(/obj/item/reagent_containers/glass/bottle/epinephrine, 5)
+		product_list += new/datum/data/vending_product(/obj/item/reagent_containers/glass/bottle/ephedrine, 3)
```
Comment (flrsh) [inline code/modules/vending/vending.dm:1154]:
```suggestion
		product_list += new/datum/data/vending_product(/obj/item/reagent_containers/glass/bottle/ephedrine, 3)
		product_list += new/datum/data/vending_product(/obj/item/reagent_containers/glass/bottle/epinephrine, 5)	
```

File: code/modules/vending/vending.dm [hunk 2]
```diff
@@ -1151,10 +1151,12 @@
...
 		product_list += new/datum/data/vending_product(/obj/item/reagent_containers/glass/bottle/filgrastim, 3)
 		product_list += new/datum/data/vending_product(/obj/item/reagent_containers/glass/bottle/heparin, 3)
 		product_list += new/datum/data/vending_product(/obj/item/reagent_containers/glass/bottle/insulin, 3)
 		product_list += new/datum/data/vending_product(/obj/item/reagent_containers/glass/bottle/morphine, 4)
+		product_list += new/datum/data/vending_product(/obj/item/reagent_containers/glass/bottle/menthol, 4)
```
Comment (flrsh) [inline code/modules/vending/vending.dm:1159]:
```suggestion
		product_list += new/datum/data/vending_product(/obj/item/reagent_containers/glass/bottle/menthol, 4)
		product_list += new/datum/data/vending_product(/obj/item/reagent_containers/glass/bottle/morphine, 4)
```

File: code/obj/machinery/porters.dm [hunk 3]
```diff
@@ -859,18 +859,20 @@ var/global/list/portable_machinery = list() // stop looping through world for th
 		product_list += new/datum/data/vending_product(/obj/item/reagent_containers/emergency_injector/charcoal, 10)
 		product_list += new/datum/data/vending_product(/obj/item/reagent_containers/emergency_injector/antihistamine, 6)
 		product_list += new/datum/data/vending_product(/obj/item/reagent_containers/emergency_injector/epinephrine, 10)
+		product_list += new/datum/data/vending_product(/obj/item/reagent_containers/emergency_injector/ephedrine, 6)
```
Comment (flrsh) [inline code/obj/machinery/porters.dm:862]:
```suggestion
		product_list += new/datum/data/vending_product(/obj/item/reagent_containers/emergency_injector/ephedrine, 6)
		product_list += new/datum/data/vending_product(/obj/item/reagent_containers/emergency_injector/epinephrine, 10)
```

**Review by flrsh — APPROVED** [review]
Seems like a fine addition, please commit my suggestions to keep them alphabetized though.

**Comment by flrsh** [comment]
Sorry for the fake merge, I discussed this with other devs and I think we want to keep these as primarily botany chems.

**Comment by NightmareChamillian** [comment]
> Sorry for the fake merge, I discussed this with other devs and I think we want to keep these as primarily botany chems.

Both menthol and ephedrine, or just menthol? Ephedrine isn't even remotely a botany chem but still has some big merits for medical use and regardless of menthol I'd still really like it to be in the nanomed.

**Pull request merged**